### PR TITLE
Update cheatsheet.md (Add --sort-by option for `top` command)

### PR DESCRIPTION
**Pull request by andrzejsydor**
Update cheatsheet.md (Add --sort-by option for `top` command)

### CODE REVIEW

**Comment by bradtopol** [comment]
Hi @andrzejsydor Next time you can just do a git commit -a and then a git push to your previous commit and that will update the PR.  You do not have to close a PR and submit a new one to make revisions to a PR.   
This is now correct syntax.

**Comment by sftim** [comment]
LGTM
@bradtopol did you want to review this one?

**Comment by bradtopol** [comment]
@sftim Yep I got it. Thanks

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -314,6 +314,7 @@ kubectl exec my-pod -- ls /                         # Run command in existing po
 kubectl exec --stdin --tty my-pod -- /bin/sh        # Interactive shell access to a running pod (1 container case) 
 kubectl exec my-pod -c my-container -- ls /         # Run command in existing pod (multi-container case)
 kubectl top pod POD_NAME --containers               # Show metrics for a given pod and its containers
+kubectl top pod POD_NAME --sort-by=cpu              # Show metrics for a given pod and sort it by 'cpu' or 'memory'
```
Comment (bradtopol) [inline content/en/docs/reference/kubectl/cheatsheet.md:317]: Looks great! Syntax is now accurate and renders properly in preview

**Comment by bradtopol** [comment]
Great job
/lgtm
/approve

**Pull request merged**